plugins/telink-dfu/telink-dfu.quirk

```diff
@@ -1,7 +1,23 @@
 # 8266 mouse beta test
 [BLUETOOTH\VID_248A&PID_8266]
 Plugin = telink_dfu
+GType = FuTelinkDfuBleDevice
+Nickname = Telink BT Mouse Prototype
 
 # 8208 Dual Keyboard
 [BLUETOOTH\VID_248A&PID_8208]
 Plugin = telink_dfu
+GType = FuTelinkDfuBleDevice
+Nickname = Telink BT Keyboard Prototype
```

Comment:
No `Telink` (it gets added automatically from the vendor), so just `Name = BT Keyboard Prototype`

Reply:
resolved in commit 29124dafd375d2e5d350abd203d833e12fb7e865
use 'Name' filed in quirk file for device name